src/archivematicaCommon/lib/archivematica_transfer_types.py

```diff
@@ -0,0 +1,127 @@
+TRANSFER_TYPE_ZIPPED_BAG = "zipped bag"
+TRANSFER_TYPE_DSPACE = "dspace"
+TRANSFER_TYPE_MAILDIR = "maildir"
+TRANSFER_TYPE_TRIM = "TRIM"
+TRANSFER_TYPE_DATAVERSE = "dataverse"
+
+TRANSFER_TYPE_DISK_IMAGE = "disk image"
+
+WATCHED_DIRECTORY_STANDARD = "standardTransfer"
+WATCHED_DIRECTORY_ZIPPED_PACKAGE = "zippedPackage"
+WATCHED_DIRECTORY_UNZIPPED_BAG = "baggitDirectory"
+WATCHED_DIRECTORY_ZIPPED_BAG = "baggitZippedDirectory"
+WATCHED_DIRECTORY_DSPACE = "Dspace"
+WATCHED_DIRECTORY_MAILDIR = "maildir"
+WATCHED_DIRECTORY_TRIM = "TRIM"
+WATCHED_DIRECTORY_DATAVERSE = "dataverseTransfer"
+
+ARCHIVE_ZIP = ".zip"
+ARCHIVE_TGZ = ".tgz"
+ARCHIVE_TAR_GZ = ".tar.gz"
+
+ARCHIVE_TYPES = (ARCHIVE_ZIP, ARCHIVE_TGZ, ARCHIVE_TAR_GZ)
+ZIP_TYPE_TRANSFERS = (
+    TRANSFER_TYPE_ZIPPED_PACKAGE,
+    TRANSFER_TYPE_ZIPPED_BAG,
+    TRANSFER_TYPE_DSPACE,
+)
+
+# Manual approval jobs associated with a transfer.
+APPROVE_STANDARD = "Approve standard transfer"
+APPROVE_ZIPPED = "Approve zipped transfer"
+APPROVE_DSPACE = "Approve DSpace transfer"
+APPROVE_UNZIPPED_BAG = "Approve bagit transfer"
+APPROVE_ZIPPED_BAG = "Approve zipped bagit transfer"
+
+APPROVE_TRANSFER_JOB_NAMES = (
+    APPROVE_STANDARD,
+    APPROVE_ZIPPED,
+    APPROVE_DSPACE,
+    APPROVE_UNZIPPED_BAG,
+    APPROVE_ZIPPED_BAG,
+)
+
+# Archivematica transfer profiles that describe a transfer's properties.
+TransferProfile = collections.namedtuple(
+    "TransferProfile", "transfer_type watched_directory"
+)
+TRANSFER_TYPES = {
+    TRANSFER_TYPE_STANDARD: TransferProfile(
+        TRANSFER_TYPE_STANDARD, WATCHED_DIRECTORY_STANDARD
+    ),
+    TRANSFER_TYPE_ZIPPED_PACKAGE: TransferProfile(
+        TRANSFER_TYPE_ZIPPED_PACKAGE, WATCHED_DIRECTORY_ZIPPED_PACKAGE
+    ),
+    TRANSFER_TYPE_UNZIPPED_BAG: TransferProfile(
+        TRANSFER_TYPE_UNZIPPED_BAG, WATCHED_DIRECTORY_UNZIPPED_BAG
+    ),
+    TRANSFER_TYPE_ZIPPED_BAG: TransferProfile(
+        TRANSFER_TYPE_ZIPPED_BAG, WATCHED_DIRECTORY_ZIPPED_BAG
+    ),
+    TRANSFER_TYPE_DSPACE: TransferProfile(
+        TRANSFER_TYPE_DSPACE, WATCHED_DIRECTORY_DSPACE
+    ),
+    TRANSFER_TYPE_MAILDIR: TransferProfile(
+        TRANSFER_TYPE_MAILDIR, WATCHED_DIRECTORY_MAILDIR
+    ),
+    TRANSFER_TYPE_TRIM: TransferProfile(TRANSFER_TYPE_TRIM, WATCHED_DIRECTORY_TRIM),
+    TRANSFER_TYPE_DATAVERSE: TransferProfile(
+        TRANSFER_TYPE_DATAVERSE, WATCHED_DIRECTORY_DATAVERSE
+    ),
+}
+
+
+def retrieve_watched_dirs():
+    """Return a generator of tuples of Archivematica Transfer Types, which
+    contains the transfer type, and suffix that will form part of the
+    transfer's watched directory.
+    """
+    for type_ in TRANSFER_TYPES:
+        yield TRANSFER_TYPES[type_].transfer_type, TRANSFER_TYPES[
+            type_
+        ].watched_directory
+
+
+def retrieve_watched_directory(transfer_type):
+    if transfer_type not in TRANSFER_TYPES:
+        return None
+    return TRANSFER_TYPES[transfer_type].watched_directory
```

Comment:
Written to be identical to any use of: 
```
TRANSFER_TYPE_DIRECTORIES = {	
    "standard": "standardTransfer",	
    "zipfile": "zippedDirectory",	
    "unzipped bag": "baggitDirectory",	
    "zipped bag": "baggitZippedDirectory",	
    "dspace": "Dspace",	
    "maildir": "maildir",	
    "TRIM": "TRIM",	
    "dataverse": "dataverseTransfer",	
}
```
in `filesystem_ajax/views.py` specifically with a view to drop-in the previous method if absolutely called for.